source/docs/romi-robot/imaging-romi.rst

```diff
@@ -3,44 +3,70 @@ Imaging your Romi
 
 Imaging
 ^^^^^^^
 
-Imaging is identical to the standard :doc:`WPILibPi Installation </docs/software/vision-processing/wpilibpi/installing-the-image-to-your-microsd-card>`.
+The procedure for installing the image is described here: :doc:`WPILibPi Installation</docs/software/vision-processing/wpilibpi/installing-the-image-to-your-microsd-card>`.
 
 Wireless Network Setup
 ^^^^^^^^^^^^^^^^^^^^^^
 
-After turning the Romi on, connect to the WiFi SSID ``WPILibPi`` with the WPA2 passphrase ``WPILib2021!``. Open a web browser and connect to either ``http://10.0.0.2/`` or http://wpilibpi.local/ to open the web dashboard.
+Perform the following steps to get your Raspberry Pi ready to use with the Romi:
+
+1. Turn the Romi on by sliding the power switch on the Romi 32U4 board to the on position. The first time it is started with a new image it will take approximately 2-3 minutes to boot while it resizes the file system and reboots. Subsequent times it will boot in less than a minute.
+
+2. Using your computer, connect to the Romi WiFi network using the SSID ``WPILibPi`` with the WPA2 passphrase ``WPILib2021!``.
+
+3. Open a web browser and connect to the Raspberry Pi dashboard at either ``http://10.0.0.2/`` or http://wpilibpi.local/.
 
 .. note:: The image boots up read-only by default, so it is necessary to click the ``Writable`` button to make changes. Once done making changes, click the ``Read-Only`` button to prevent memory corruption.
 
-It is very important to change the default password associated with your Romi. Select ``Writable`` at the top of the page then set a new password in the ``WPA2 Passphrase`` field. Press the ``Save`` button at the bottom to save changes. You will need to reconnect to the Romi's WiFi network with the new password.
+4. Select ``Writable`` at the top of the dashboard web page.
+
+5. Change the default password for your Romi by setting a new password in the ``WPA2 Passphrase`` field.
+
+6. Press the ``Save`` button at the bottom of the page to save changes.
+
+7. Change the network SSID to a unique name if you plan on operating your Romi on wireless network with other Romis.
+
+8. Reconnect to the Romi's WiFi network with the new password you set.
+
+Be sure to set the Dashboard to ``Read-only`` when all the changes have been completed.
 
 .. image:: images/imaging-romi/network-settings.png
    :alt: Romi web dashboard network settings
 
 32U4 Control Board
 ------------------
 
-With the Raspberry Pi now imaged, we can use it to image the 32U4 Control Board. Before turning the Romi on, use a USB A to micro-B cable to connect one of the Raspberry Pi's USB ports to the micro USB port on the 32U4 Control Board. Next, turning the Romi on, connect to its Wifi network, open a web browser, and connect to the web dashboard. On the ``Romi`` configuration page, press the ``Update Firmware`` button.
+The Raspberry Pi can now be used to write the firmware image to the 32U4 Control Board.
+
+#. Turn off the Romi
```

Comment:
Should these just be
- Turn off the Romi

Reply:
I am not sure what you are asking

Reply:
You are using ``#.`` and not ``-``

Reply:
But it is an ordered step.

Reply:
Is there supposed to be a 1. 2. 3. 4. I'm confused?

Reply:
There is. 
![image](https://user-images.githubusercontent.com/4885091/102699388-1d598400-41f9-11eb-8c7a-04ed33d5d24c.png)
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#ordered-lists